example/src/main/java/org/aerogear/mobile/example/ui/SecurityServiceFragment.java

```diff
@@ -154,14 +147,6 @@ public void detectEmulator(Map<String, SecurityCheckResult> results) {
         }
     }
 
-    /**
-     * Detect if a hooking framework application is installed on the device
-     */
-    public void detectHookingFramework(Map<String, SecurityCheckResult> results) {
```

Comment:
Do we intend on adding this check at all or is it not being added?

Reply:
@ciaranRoche @aidenkeating I think we can drop this one for now. It was really a POC for the mobile security project.